app/src/main/java/org/raleighmasjid/iar/composable/donate/DonateScreen.kt

```diff
@@ -57,6 +61,16 @@ fun DonateScreen() {
                 modifier = Modifier.padding(vertical = 8.dp))
         }
     }
+
+    if (onClickLaunchPage.value) {
+        onClickLaunchPage.value = false
+        LaunchDonatePage()
+    }
+}
+
+@Composable fun LaunchDonatePage() {
+    val uriHandler = LocalUriHandler.current
+    uriHandler.openUri("https://raleighmasjid.org/donate/")
```

Comment:
can we put this code directly in the `onClick` closure of the button?

Something like 

```
OutlinedButton(
             onClick = {
                 val uriHandler = LocalUriHandler.current
                  uriHandler.openUri("https://raleighmasjid.org/donate/")
             }
```

Reply:
Unfortunately not, I keep getting an error that I can't call a composable function inside a nonComposable function (onClick), which is why I had to take this weird roundabout route

Reply:
so I think we just need to declare the handler at the top of the `DonateScreen` composable.

```
@Composable
fun DonateScreen() {
    val uriHandler = LocalUriHandler.current

    Column(
```

and then in the `onClick` we can call openUri

```
OutlinedButton(
            onClick = {
                uriHandler.openUri("https://raleighmasjid.org/donate/")
            },
```

Reply:
Yeah that works, I just updated the code and pushed it